### PR TITLE
Frozen artifacts

### PR DESCRIPTION
**Pull request by arithon**
## Changelog:
- Added Deathwhisper for Sylvanas Windrunner.
- Added Serathil for Kor'Gall.
- Distribution of first paladin librams, Silverhand tabards, Hammer of the Lightbringer.
- Distribution of Frostmourne.
- Distribution of Warblade of Archimonde.
- Added Orb of Kil'Jaeden for Illidan.
- Added Scourge Bone Chimes for Arthas and some random landed scourge characters.
- Added Quel'delar for Lana'thel.

### CODE REVIEW

**Comment by sejtalk** [comment]
will make new deathwhisper icon tommorow

File: common/artifacts/901_wc_artifacts_weapon_onehanded.txt [hunk 1]
```diff
@@ -53,4 +53,33 @@ hammer_of_the_twilight = {
...
+	quality = 3
+	flags = { combat_weapon axe }
+
+	active = {
+		is_adult = yes
+		OR = {
+			NOT = { has_class_trait_trigger = yes }
+			is_physical_class_trigger = yes
+			is_combined_class_trigger = yes
+		}
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifacts/901_wc_artifacts_weapon_onehanded.txt]: Ok, this was from before that trigger was a thing, I think. Done

File: common/artifacts/901_wc_artifacts_weapon_onehanded.txt [hunk 2]
```diff
@@ -53,4 +53,33 @@ hammer_of_the_twilight = {
...
+	}
+	allowed_gift = {
+		always = no
+	}
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifacts/901_wc_artifacts_weapon_onehanded.txt]: My bad, another remnant from earlier times ;) Done

File: common/artifacts/901_wc_artifacts_weapon_twohanded.txt [hunk 3]
```diff
@@ -434,6 +434,32 @@ hammer_of_the_lightbringer = {
 
 	slot = weapon
 }
+deathwhisper = {
+	quality = 4
+	flags = { combat_weapon bow }
+
+	active = {
+		shared_physical_weapon_trigger = yes
+	}
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifacts/901_wc_artifacts_weapon_twohanded.txt]: Well, it's a bow... I don't really see a reason to exclude paladins from it...

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 4]
```diff
@@ -73,7 +73,7 @@ gauntlets_of_trollbanes = {
 			value = 100
 			has_landed_title = k_stromgarde
 		}
-		# If Stromgarde is failed, gives to random humans
+		# If Stromgarde  failed, gives to random humans
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt]: Overlooked it, done.

File: common/artifacts/901_wc_artifacts_weapon_onehanded.txt [hunk 5]
```diff
@@ -78,4 +78,27 @@ ironfoe = {
...
 }
+serathil = {
+	quality = 3
+	flags = { combat_weapon axe }
 
+	active = {
+		is_adult = yes
+		OR = {
+			shared_physical_weapon_trigger = yes
+		}
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifacts/901_wc_artifacts_weapon_onehanded.txt:89]: done

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 6]
```diff
@@ -973,4 +972,129 @@ warblade_of_archimonde = {
...
+		additive_modifier = {
+			value = 100
+			character = 340009						# Illidan
+		}
+	}
+	artifacts = {
+		orb_of_kiljaeden = { value = 1 }
+	}
+}
+scourge_bone_chimes  = {
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt:995]: fixed

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 7]
```diff
@@ -973,4 +972,129 @@ warblade_of_archimonde = {
 	artifacts = {
 		warblade_of_archimonde = { value = 1 }
 	}
+}
+orb_of_kiljaeden = {
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt:976]: fixed

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 8]
```diff
@@ -973,4 +972,141 @@ warblade_of_archimonde = {
...
+		legion_doomhorn = { value = 1 }
+	}
+}	
+serathil = {
+	max_amount = 1
+	spawn_date = 605.6.1
+	
+	spawn_chance = {
+		value = 100
+	}
```
Comment (zumbak04) [inline common/artifact_spawns/wc_artifact_spawns.txt:1057]: The same problem. Spawns before the spawn date.

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 9]
```diff
@@ -973,4 +972,141 @@ warblade_of_archimonde = {
...
+		serathil = { value = 1 }
+	}
+}
+deathwhisper = {
+	max_amount = 1
+	spawn_date = 605.6.1
+	
+	spawn_chance = {
+		value = 100
+	}
```
Comment (zumbak04) [inline common/artifact_spawns/wc_artifact_spawns.txt:1076]: The same problem. Spawns before the spawn date.

File: common/artifact_spawns/wc_artifact_spawns.txt (hunk 8 above)
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt:1057]: fixed

File: common/artifact_spawns/wc_artifact_spawns.txt (hunk 9 above)
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt]: fixed

**Pull request merged**